src/dat/sys/routes.clj

```diff
@@ -0,0 +1,44 @@
+(ns dat.sys.routes
```

Comment:
I split out the base ring-handler from the fallback routes. I'm not familiar with bidi so i made a stub routes.cljc and just used some clj only compojure to test the routes were working properly.